core/src/main/java/com/nukesz/game/OnscreenControlRenderer.java

```diff
@@ -0,0 +1,46 @@
+package com.nukesz.game;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.graphics.g2d.TextureRegion;
+
+public class OnscreenControlRenderer {
+    SpriteBatch batch;
+    TextureRegion left;
+    TextureRegion right;
+    //Texture up;
+    TextureRegion down;
+    TextureRegion up;
+
+    public OnscreenControlRenderer() {
+        loadAssets();
+    }
+
+    private void loadAssets() {
+        Texture texture = new Texture(Gdx.files.internal("sidearrow.png"));
+        Texture texture2 = new Texture(Gdx.files.internal("topbottonarrow.png"));
+        TextureRegion[] buttons = TextureRegion.split(texture, 30, 480)[0];
+        TextureRegion[] buttons2 = TextureRegion.split(texture2, 640, 30)[0];
```

Comment:
I think it would be better to use TextureAtlas here. (maybe merge the two files together also.

Reply:
I would like to do that in another patch.